Server/Views/Home/Search.cshtml

```diff
@@ -1,39 +1,49 @@
 ﻿@model IEnumerable<Common.Data.TermDoc>
 
 @{
-    ViewData["Title"] = "Search";
+    ViewData["Title"] = "Search Result";
 }
 
 <h1>Search</h1>
 
-<table class="table">
+<table class="table table-striped table-bordered" id="searchResult">
     <thead>
         <tr>
-            
-            <th>
-                @Html.DisplayNameFor(model => model.DocumentPath)
-            </th>
+
             <th>
                 @Html.DisplayNameFor(model => model.LineNumber)
             </th>
             <th>
                 @Html.DisplayNameFor(model => model.LinePosition)
             </th>
+            <th>
+                @Html.DisplayNameFor(model => model.DocumentPath)
```

Comment:
Why do you want the path to be the last column? Isn't it the most important/interesting column?

Reply:
The user dont have to see the path, they just need something to open the document. Show we actually dont need to show the path for them

Reply:
But they can't open the document yet. But ok.